### DISCUSSION TITLE
Beispiel und TaxCategoryCodes.

### DISCUSSION THREAD

**Discussion by Otto404** [stephanstapel/ZUGFeRD-csharp#375] [Q&A] [answered]
ich habe mir eben noch einmal das Beispiel angesehen. (Anlegen einer Rechnung)
und dort finde ich:
desc.AddApplicableTradeTax(129.37m, 7m, TaxTypes.VAT, TaxCategoryCodes.S);
desc.AddApplicableTradeTax(64.46m, 19m, TaxTypes.VAT, TaxCategoryCodes.S);

müsste es nicht richtig heißen:
desc.AddApplicableTradeTax(129.37m, 7m, TaxTypes.VAT, TaxCategoryCodes.AA);
desc.AddApplicableTradeTax(64.46m, 19m, TaxTypes.VAT, TaxCategoryCodes.S);

**Comment by stephanstapel** [comment, marked as answer]
aus den offiziellen Beispielen (auch eine gute Informationsquelle):

![grafik](https://github.com/user-attachments/assets/04fe34df-047f-48e5-91ae-6c123cfd5189)

**Reply by Otto404** [reply]
Danke. Also "offiziell" auch beim reduzierten Steuersatz die "S". Aber wofür gibt es nun:
    // Zusammenfassung:
    //     Lower rate Tax rate is lower than standard rate.
   TaxCategoryCodes.AA,

wo setzte ich "AA" ein?

**Reply by stephanstapel** [reply]
das weiß ich leider auch nicht. Selbst in der temporären Umstellung der MwSt-Sätze während der Corona-Pandemie hätte man ja vermutlich 'S' genutzt. Es gibt analog umgekehrt übrigens auch H für die Higher Rate.

**Reply by Otto404** [reply]
Beim Lesen der Originale läuft mir Honig aus dem Ohr und ich weiß nicht mehr, ob ich Männlein oder Weiblein bin. Schon die Redundanz mit dem Gesamtsummen (und was ist was?) erhöht den Stress. Ich hatte mit dem XSD angefangen und habe mich über die Freude Deinen Code gefunden zu haben, schier überschlagen. 
Vielleicht kannst Du ja noch etwas zu meiner anderen Frage sagen?
